amr-wind/wind_energy/actuator/wing/ActuatorWing.H

```diff
@@ -37,6 +37,15 @@ struct WingBaseData
     //! Pitch angle for the wing
     amrex::Real pitch{0.0};
 
+    //! File name for pitch actuation table
+    std::string pitch_timetable;
```

Comment:
Let's add an identifier to the variable name, this would be best: "pitch_timetable_file"

Reply:
Done